codegen/src/main/kotlin/com/android/designcompose/codegen/BuilderProcessor.kt

```diff
@@ -599,30 +578,28 @@ class BuilderProcessor(private val codeGenerator: CodeGenerator, val logger: KSP
             function: KSFunctionDeclaration,
             data: Unit,
             nodeName: String,
-            override: Boolean
         ) {
             // Generate the function name and args
             out.appendText("    @Composable\n")
-            val overrideKeyword = if (override) "override " else ""
-            out.appendText("    ${overrideKeyword}fun $function(\n")
+            out.appendText("    final fun $function(\n")
 
             // Collect arguments into a list so we can reuse them
             val args: ArrayList<Pair<String, String>> = ArrayList()
 
-            // Add a modifier for the root item. Default params not allowed with override
-            val defaultModifier = if (override) "" else " = Modifier"
+            // Add a modifier for the root item.
+            val defaultModifier = " = Modifier"
             args.add(Pair("modifier", "Modifier$defaultModifier"))
 
             // Add an option to register a function to handle open link callbacks
-            val defaultOpenLink = if (override) "" else " = null"
+            val defaultOpenLink = " = null"
             args.add(Pair("openLinkCallback", "OpenLinkCallback?$defaultOpenLink"))
```

Comment:
Can we just combine these two lines (and the others like it?)

Reply:
yep, meant to do that.